model/host/host.go

```diff
@@ -1794,30 +1814,28 @@ func (hosts HostGroup) Uphosts() HostGroup {
 // getNumContainersOnParents returns a slice of uphost parents and their respective
 // number of current containers currently running in order of longest expected
 // finish time
-func GetNumContainersOnParents(d distro.Distro) ([]ContainersOnParents, error) {
+func GetContainersOnParents(d distro.Distro) ([]ContainersOnParents, error) {
```

Comment:
I don't think this will scale. I think it's probably fine, but we should also open a ticket to do this all in an aggregation/lookup?

Reply:
opened https://jira.mongodb.org/browse/EVG-7405